src/journey-maps/angular/services/constants.ts

```diff
@@ -11,10 +11,13 @@ export const SBB_BOUNDING_BOX: LngLatBoundsLike = [
   [10.6677, 47.9163],
 ];
 
-export const SBB_MARKER_SOURCE = 'rokas-marker-source';
-export const SBB_ROUTE_SOURCE = 'rokas-route-source';
-export const SBB_WALK_SOURCE = 'rokas-walk-source';
-export const SBB_ZONE_SOURCE = 'rokas-zone-source';
+export const JOURNEY_POIS_SOURCE = 'journey-pois-source';
+export const ROKAS_MARKER_SOURCE = 'rokas-marker-source';
+export const ROKAS_ROUTE_SOURCE = 'rokas-route-source';
+export const ROKAS_STATION_HOVER_SOURCE = 'rokas-station-hover-source'; // TODO cdi ROKAS-1204 where is this source used ?
```

Comment:
Remove this TODO. The station source is used here: SbbMapStationService._updateStationSource

This is neccessary to show the blue circle when hovering a station in CASA.